### DISCUSSION TITLE
文章比较长的时候，目录能不能设计成跟随移动（如csdn是锁死固定出现在屏幕的某个位置）？

### DISCUSSION THREAD

**Discussion by W01fh4cker** [ppoffice/hexo-theme-icarus#1098] [Ideas]
![image](https://user-images.githubusercontent.com/101872898/181174430-2701ba6f-87bf-400d-9b01-ff3f68cc2fa4.png)
比如我这个文章6658个字，在这里的时候目录显示在这；我稍微往下划一点，可以展开二级目录：
![image](https://user-images.githubusercontent.com/101872898/181174575-437322aa-8d73-4fa1-97a5-e2bf71f8a371.png)
我再往下滑，目录就看不到了：
![image](https://user-images.githubusercontent.com/101872898/181174709-f27bb838-8fc7-428d-8cda-c4f6da6ebd94.png)
这里感觉可以借鉴一下csdn的解决方法，让目录一直呆在左下角：
![image](https://user-images.githubusercontent.com/101872898/181174831-e89c5e47-615c-4fcd-b148-847f55142b94.png)

**Comment by ppoffice** [comment]
https://ppoffice.github.io/hexo-theme-icarus/Configuration/icarus%E7%94%A8%E6%88%B7%E6%8C%87%E5%8D%97-%E4%B8%BB%E9%A2%98%E9%85%8D%E7%BD%AE/#%E4%BE%A7%E8%BE%B9%E6%A0%8F

**Reply by W01fh4cker** [reply]
十分感谢，我之前搜关键词目录没看到就以为没有，原来是这样。😂😂

**Reply by bbuugg** [reply]
> https://ppoffice.github.io/hexo-theme-icarus/Configuration/icarus%E7%94%A8%E6%88%B7%E6%8C%87%E5%8D%97-%E4%B8%BB%E9%A2%98%E9%85%8D%E7%BD%AE/#%E4%BE%A7%E8%BE%B9%E6%A0%8F

![image](https://user-images.githubusercontent.com/64066545/193731754-6dda6fcd-f1d1-4514-87b6-0d8e0c288f75.png)

如图，我的目录太长，导致即使是sticky下面的也看不见，而且目录不能滚动，超出需要滑动到最底端才能看到，但是目录和当前文章位置不能对应起来。请问有什么配置可以更改下吗。
】

**Reply by W01fh4cker** [reply]
我之前目录内容比较少的时候，没有出现问题，所以看到了但是没有回复你。目录内容多了之后，我也出现了这个问题，我`windows`上面的浏览器看会出现显示不全的问题，而我的`ubuntu`的浏览器没有问题，需要我手动调整页面的大小才可以完全显示，希望这里可以优化一下。十分感谢作者`ppoffice`的辛苦付出。
